src/python/serenity_types/pricing/derivatives/options/volsurface.py

```diff
@@ -162,11 +160,10 @@ class InterpolatedVolatilitySurface(BaseModel):
     All fitted vols, the z-axis in the mesh.
     """
 
-    option_types: List[OptionType]
+    calibration_params: Dict[AnyStr, AnyStr]
     """
-    Due to dirty data on Deribit, our calibration fits separately for PUT and CALL options.
-    To allow pricing of either option type, the fitted surface returns both. This array
-    tells you the type for the corresponding fitted volatility in the vols array.
+    Informational set of calibration paramters, e.g. the SVI parameters. May be empty and
+    keys will depend on the VolModel.
     """
```

Comment:
@indra-cloudwall this is the only scope addition (rest are just docstring changes or subtractions) but I think it's a trivial one. Feel free to leave empty for version 1 if it's a hassle.